ezgui/src/svg.rs

```diff
@@ -12,8 +12,8 @@ pub const LOW_QUALITY: f32 = 1.0;
 // Code here adapted from
 // https://github.com/nical/lyon/blob/0d0ee771180fb317b986d9cf30266722e0773e01/examples/wgpu_svg/src/main.rs
 
-pub fn load_svg(prerender: &Prerender, filename: &str, scale_factor: f64) -> (GeomBatch, Bounds) {
```

Comment:
mapspace and screenspace svg loading code now do the same thing, so I consolidated them into a single method.

Furthermore, there was no longer currently a use for any scale_factor for load_svg. This *might* be controversial. I could imagine scaling SVG's differently depending on where we want them in the UI. Happy to revert this portion and have everything currently using is plumb through a `scale_factor: 1.0` if you'd prefer.

Reply:
This is a good change; scale factor being part of the cache key was a headache. If other code wants something scaled up, it can call `.scale` on the geombatch